docs/source/en/model_doc/vitpose.md

```diff
@@ -1,286 +1,151 @@
-person_boxes[:, 3] = person_boxes[:, 3] - person_boxes[:, 1]
-
-# ------------------------------------------------------------------------
-# Stage 2. Detect keypoints for each person found
-# ------------------------------------------------------------------------
-
-image_processor = AutoProcessor.from_pretrained("usyd-community/vitpose-base-simple")
-model = VitPoseForPoseEstimation.from_pretrained("usyd-community/vitpose-base-simple", device_map=device)
-
-inputs = image_processor(image, boxes=[person_boxes], return_tensors="pt").to(device)
+# Step 2: Run ViTPose on the cropped person(s)
+pose_processor = AutoProcessor.from_pretrained("usyd-community/vitpose-base-simple")
+pose_model = AutoModelForPoseEstimation.from_pretrained("usyd-community/vitpose-base-simple").to("cuda")
 
+pose_inputs = pose_processor(image, boxes=[person_boxes], return_tensors="pt").to("cuda")
 with torch.no_grad():
-    outputs = model(**inputs)
+    pose_outputs = pose_model(**pose_inputs)
 
-pose_results = image_processor.post_process_pose_estimation(outputs, boxes=[person_boxes])
-image_pose_result = pose_results[0]  # results for first image
+pose_results = pose_processor.post_process_pose_estimation(pose_outputs, boxes=[person_boxes])
 ```
 
-### ViTPose++ models
+</hfoption>
+</hfoptions>
 
-The best [checkpoints](https://huggingface.co/collections/usyd-community/vitpose-677fcfd0a0b2b5c8f79c4335) are those of the [ViTPose++ paper](https://arxiv.org/abs/2212.04246). ViTPose++ models employ a so-called [Mixture-of-Experts (MoE)](https://huggingface.co/blog/moe) architecture for the ViT backbone, resulting in better performance.
+---
 
-The ViTPose+ checkpoints use 6 experts, hence 6 different dataset indices can be passed. 
-An overview of the various dataset indices is provided below:
+## Quantization Example
 
-- 0: [COCO validation 2017](https://cocodataset.org/#overview) dataset, using an object detector that gets 56 AP on the "person" class
-- 1: [AiC](https://github.com/fabbrimatteo/AiC-Dataset) dataset
-- 2: [MPII](https://www.mpi-inf.mpg.de/departments/computer-vision-and-machine-learning/software-and-datasets/mpii-human-pose-dataset) dataset
-- 3: [AP-10K](https://github.com/AlexTheBad/AP-10K) dataset
-- 4: [APT-36K](https://github.com/pandorgan/APT-36K) dataset
-- 5: [COCO-WholeBody](https://github.com/jin-s13/COCO-WholeBody) dataset
-
-Pass the `dataset_index` argument in the forward of the model to indicate which experts to use for each example in the batch. Example usage is shown below:
+```bash
+pip install torchao
+```
 
 ```python
-image_processor = AutoProcessor.from_pretrained("usyd-community/vitpose-plus-base")
-model = VitPoseForPoseEstimation.from_pretrained("usyd-community/vitpose-plus-base", device=device)
-
-inputs = image_processor(image, boxes=[person_boxes], return_tensors="pt").to(device)
-
-dataset_index = torch.tensor([0], device=device) # must be a tensor of shape (batch_size,)
+import torch
```

Comment:
You can use the `TorchAoConfig` here instead, see the [Gemma 3](https://huggingface.co/docs/transformers/main/en/model_doc/gemma3#gemma-3) for an example